src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -16,11 +16,19 @@ def load_command_table(self, _):
         operations_tmpl='azext_connectedk8s.vendored_sdks.operations#ConnectedClusterOperations.{}',
         client_factory=cf_connectedk8s
     )
-
+    connectedk8s_sdk_prev = CliCommandType(
+        operations_tmpl='azext_connectedk8s.vendored_sdks.preview_2021_04_01.operations#ConnectedClusterOperations.{}',
+        client_factory=cf_connectedk8s_prev_2021_04_01
+    )
     with self.command_group('connectedk8s', connectedk8s_sdk, client_factory=cf_connected_cluster) as g:
         g.custom_command('connect', 'create_connectedk8s', supports_no_wait=True)
         g.custom_command('update', 'update_agents')
         g.custom_command('upgrade', 'upgrade_agents')
         g.custom_command('delete', 'delete_connectedk8s', confirmation=True, supports_no_wait=True)
+        g.custom_command('enable-features', 'enable_features', is_preview=True)
+        g.custom_command('disable-features', 'disable_features', is_preview=True)
         g.custom_command('list', 'list_connectedk8s', table_transformer=connectedk8s_list_table_format)
         g.custom_show_command('show', 'get_connectedk8s', table_transformer=connectedk8s_show_table_format)
+
+    with self.command_group('connectedk8s', connectedk8s_sdk_prev, client_factory=cf_connected_cluster_prev_2021_04_01) as g:
+        g.custom_command('proxy', 'client_side_proxy_wrapper', is_preview=True)
```

Comment:
>client_side_proxy_wrapper [](start = 35, length = 25)

What is this string for?

Reply:
this is the function that gets invoked for az connectedk8s proxy. its in a different command group because it is using preview sdk.

Reply:
this is the name of the function which is called for that az proxy command. @atharvamulmuley right?